theme/less/_content.less

```diff
@@ -1,4 +1,62 @@
-main article img
+article
+{
+  font-size: 1rem;
+  color: #444444;
+  font-weight: 400;
+  line-height: 1.5;
+}
+
+article img
 {
   max-width: 100%;
 }
+
+article p:first-of-type img
+{
+  margin-top: 1em;
+}
+
+article h1
```

Comment:
My system doesn't have "PT Sans" installed and fonts are rendered differently on for me in comparison to the Drupal version of the website, that's seem to be because in Drupal version Fonts are being requested from [https://fonts.google.com/](https://chrome-automation.com/themes/skeletontheme/css/style.css).

We can either fix font issue in current issue or separately. To fix it, I think first we need to ask the questions:
- Do we want to use PT Sans?
- How we want to import them?

## Using PT Sans

Using PT Sans sounds like a low hanging fruit and we can definately use it. We should for future take into consideration that it's only supported in `cyrillic` and `latin`. In general fonts and multi-language support is a quite complex topic, I have briefly touch that in [current talk - from 10:45min](https://youtu.be/YpRc79o-QHM?t=645).

Usually when I decide about fonts I open https://fonts.google.com/ and sort them by popularity, Roboto, Source Sans Pro and Open Sans, are those I was working lately, it's a matter of choice and preference, thinking it's best to find one matching closest to PT Sans, or just go with PT Sans. and call it a day.

## Importing fonts

We should ensure that the font we are using is woff2 format, as it's currently most advanced compression for those. Also it's always a good idea to host fonts on your own server rather than on Google's, that also might help with the privacy concenrs, because if fonts are downloaded from https://fonts.google.com, then we basically sending request to another server. In most of the cases it's not a problem, but regarding the privacy I think if you can avoid then you should.

Here are couple of examples of fonts implementation with CMintS:
- [Multilanguage](https://github.com/cmints/website/blob/master/theme/less/_fonts.less)
- [Single language](https://github.com/Manvel/blog/blob/master/theme/less/fonts.less)

For fonts conversion I guess bet tool would be [font-tools](https://github.com/fonttools/fonttools), but usually you can get ready woff2 fonts from https://fonts.google.com, I guess in the video above there are also solutions for that.